package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "npx nx dev server & npx nx dev client"
+    "dev": "run-p server client",
+    "server": "npx nx dev server",
+    "client": "npx nx dev client"
   },
   "private": true,
-  "dependencies": {},
   "devDependencies": {
     "@nx/js": "16.8.1",
+    "npm-run-all": "^4.1.5",
```

Comment:
@metehancelik  I think you don't need an external package to run apps in paralles. Nx also comes with a handy command to achieve this. Please remove this package and try the following command.
`npx nx run-many --target=dev --projects=server,client`

Reply:
i tried this command but --target option was wrong. fixed it now.